scene/resources/visual_shader.cpp

```diff
@@ -1466,6 +1466,17 @@ bool VisualShader::is_text_shader() const {
 	return false;
 }
 
+#ifndef DISABLE_DEPRECATED
+void VisualShader::set_graph_offset(const Vector2 &p_offset) {
+	WARN_DEPRECATED_MSG("graph_offset property is deprecated. Setting it has no effect.");
+}
+
+Vector2 VisualShader::get_graph_offset() {
```

Comment:
This was `const`:
```suggestion
Vector2 VisualShader::get_graph_offset() const {
```

Reply:
https://github.com/godotengine/godot/actions/runs/15876950870/job/44766829909